quickstart-docker/resources/kafkafeeder.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+timestamp_now=$(date +%s)
+random_3char_suffix=$(openssl rand -base64 3)
+random_enum_index=$(($RANDOM %3))
+declare -a myArray=("GO_RIDE" "GO_SEND" "GO_SHOP")
```

Comment:
also can you give different names here of products not gojek products

Reply:
fixed